src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, partnerName, isMarried } = person;
+  const SEX_FEMALE = 'f';
+
+  return (
+    <>
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+        {age && <p className="Person__age">{`I am ${age}`}</p>}
+        <p className="Person__partner">
```

Comment:
```suggestion
        <h2 className="Person__name">{`My name is ${name}`}</h2>
        
        {age && (
          <p className="Person__age">
            {`I am ${age}`}
          </p>
        )}
        
        <p className="Person__partner">
```
Pay attention to the formatting